refinery/core/models.py

```diff
@@ -1129,7 +1129,7 @@ def delete(self, **kwargs):
     def get_status(self):
         return self.status
 
-    def get_auxiliary_nodes(self):
+    def get_auxiliary_node_uuids(self):
```

Comment:
This method belongs to `Analysis` model (not `Node`) and should not be renamed. Please check the rest of the changes to make sure only the correct function is renamed.